src/applications/personalization/profile/reducers/vaProfile.js

```diff
@@ -114,8 +120,14 @@ function vaProfile(state = initialState, action) {
       return set('eduPaymentInformationUiState.responseError', null, newState);
     }
 
-    case EDU_PAYMENT_INFORMATION_SAVE_STARTED:
-      return set('eduPaymentInformationUiState.isSaving', true, state);
+    case EDU_PAYMENT_INFORMATION_SAVE_STARTED: {
+      const eduPaymentInformationUiState = {
+        ...state.eduPaymentInformationUiState,
+        responseError: null,
```

Comment:
In writing my tests I discovered this bug, where we did not clear existing save errors when starting a new save operation.